src/site/topic23.rst

```diff
@@ -214,6 +214,38 @@ Linked Binary Search Tree
 Degenerate vs. Balanced
 =======================
 
+* A *balanced* tree has the property that for any node in the tree, the height of its left and right subtrees can differ by **at most** 1
+    * Remember, the height of an empty tree is -1
```

Comment:
Talked about in #295, this height definition doesn't really make sense. Also, [the notes say otherwise](https://github.com/jameshughes89/cs102/blame/main/src/site/topic21.rst#L122), height of the empty tree is 0. The balanced tree definition doesn't change.

Reply:
damn good catch dude.